api/src/main/java/io/opentelemetry/trace/Span.java

```diff
@@ -406,6 +407,23 @@
      */
     Builder setParent(SpanContext remoteParent);
 
+    /**
+     * Sets the parent to use from the specified {@code Context}. If not set, the value of {@code
+     * Tracer.getCurrentSpan()} at {@link #startSpan()} time will be used as parent.
+     *
+     * <p>If no {@link Span} is available in the specified {@code Context}, the resulting {@code
+     * Span} will become a root instance, as if {@link #setNoParent()} had been called.
+     *
+     * <p>If called multiple times, only the last specified value will be used. Observe that the
+     * state defined by a previous call to {@link #setNoParent()} will be discarded.
+     *
+     * @param context the {@code Context}.
+     * @return this.
+     * @throws NullPointerException if {@code context} is {@code null}.
+     * @since 0.5.0
```

Comment:
0.7.0

Reply:
I think there was a race condition between me pushing the changes and your review ;) In any case, all issues have been addressed.